db-commons/src/main/resources/reference.conf

```diff
@@ -11,8 +11,11 @@ bitcoin-s {
       //https://scala-slick.org/doc/3.3.1/api/index.html#slick.jdbc.JdbcBackend$DatabaseFactoryDef@forConfig(String,Config,Driver,ClassLoader):Database
       path = ${bitcoin-s.datadir}/${bitcoin-s.network}/
       driver = org.sqlite.JDBC
+      driverName = sqlite
```

Comment:
The driver name is a part of JDBC URL, why do we need a separate parameter for it?